### ISSUE TITLE
WALS languoid ID mappings

### ISSUE THREAD

**Issue by HedvigS** [clld/wals-data#157]
I don't know if this is useful, but I noticed these things in the WALS "language.csv" download file:

Name of huaa1248 in the language.csv download file is "Err:520", it should be "=|Hoan" or "Amkoe"

There were some languoid entries that lacked glottcode and/or iso code. I think these are the appropriate matchings in those cases:

| Name| wals_code | iso_code | Glottocode | Comment |
|:-----------------------------------------------------------|:----------|:--------------------|:---------------------------------------------------------------------------------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| Arapesh (Abu) | aab |aah |abua1245||
| Ayomán| ayo|   |ayom1234| |
| Berber (Figuig) | bfg |  grr |tasn1238| |
| Johari | joh | rgk | rang1266| |
| Juat |jua | nys| nyun1247| |
| Kenyah (Uma' Lung) | keu | ulu| umal1238 | |
| Kriol (Fitzroy Corssing) | kfc | rop| krio1252 | Ideally it'd be better to find the exact dialect code |
| Mongol (Khamnigan) |mkh | | kham1281| |
| Mixe (Ayutla) |mxx |mxp |tlah1239| |
| Nahuatl (Huauchinango) | nhu |ncj | nort2957 | |
| Nahuatl (Milpa Alta) | nmp |nhm |more1259| |
| Kualan |kua |sdm |sema1269| |
| Russian-Chinese Pidgin (Birobidjan) | rcp | | kjac1234 | |
| Romani (Sepecides) | rse | | balk1252 | |
| Tommo So | tms | | tomm1242 | |
| Tasmanian (Oyster Bay to Pitwater) | toy | |brun1235 | a bit unsure |
| Tupi | tup | pog |poti1237| |

These ones I'm unsure about.

| Name| wals_code | Comment |
|:----------------------|:---------------------------------------------------------------------------------------------------------------------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| Lughat al-Isharat al-Lubnaniya|lgh | Lebanese sign, doesn't have glottocode of its own. Maybe should be mapped to Jordanian Sign Language?|
| Tasmanian |tsm | Very unclear which Tasmanian language this should be mapped to|

**Comment by xrotwang** [comment]
The name for [=|Hoan](https://wals.info/languoid/lect/wals_code_huc) appears correctly in `language.csv`:
```bash
$ grep huaa language.csv 
huc,huc,huaa1248,=|Hoan,-25.5,25.0,=|Hoan,Kxa,Africa,BW,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,2 SVO,1 SV,2 VO,,4 No dominant order,1 Genitive-Noun,2 Noun-Adjective,2 Noun-Demonstrative,2 Noun-Numeral,1 Noun-Relative clause,,,2 Not initial interrogative phrase,,5 Other,4 VO and NRel,4 VO and NAdj,,,,,,,,,,,,,,,,,,,1 Question particle,,,,,,,,,,,,,,,,,,,,,,,,,,,4 None,,,1 Noun-Relative clause (NRel) dominant,,7 No SVNegO,,,,2 SNegVO,,,,1 Word&NoDoubleNeg,,,,,,4 No SVONeg,3 Immed preverbal,,,,2 SNegVO,,1 NegV,,,1 NegV,,,,,4 No NegSVO,,,4 None,,,,,,,,,,,,
```
What you see is the result of [LibreOffice trying to be smart and interpret the name as a formula](https://ask.libreoffice.org/en/question/31296/what-does-err520-mean-in-a-cell/), because it starts with `=`.

**Comment by HedvigS** [comment]
@xrotwang Right, that makes sense. I opened the file in Libreoffice to adjust some of the codes and then when I saved Libreoffce must have saved that error because it thought it was a formula, like you say, and then when I read it in with R I get the error string. Sigh, annoying, but understandable.

**Comment by xrotwang** [comment]
I forwarded the proposed corrections to Matthew.
